DevOps.Util.DotNet/HelixServer.cs

```diff
@@ -0,0 +1,104 @@
+﻿#nullable enable
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DevOps.Util.DotNet
+{
+    public class HelixServer : BaseServer
+    {
+        public HelixServer()
+            : base((string?)null) { }
+
+        private HelixRequestBuilder GetBuilder(string apiPath) => new HelixRequestBuilder(apiPath);
+
+        public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, string downloadDir)
+        {
+            if (!Path.IsPathFullyQualified(downloadDir))
+            {
+                downloadDir = Path.Combine(Environment.CurrentDirectory, downloadDir);
+            }
+
+            JobDetails jobDetails = await GetJsonAsync<JobDetails>(GetBuilder($"/jobs/{jobId}/details/"));
```

Comment:
Curious: why did you prefer hand calling the API vs. using the Helix libraries that wrap the REST API? I've waffled back and froth on this before. Wanted your take.

Reply:
> using the Helix libraries that wrap the REST API

Which are these libraries? Could you point them to me? I use the REST API since it is very simple to do the get request, it aligns with what we do for AzDo requests and the json responses were pretty simple -- but I could change it if we think it would be cleaner to just use the Helix libraries (that way we wouldn't need to use a client ourselves nor build the queries either).

Reply:
Know the APIs exist in arcade. Not sure if they exist in NuPkg form. Let me check with Chris.

Reply:
https://dnceng.visualstudio.com/public/_packaging?_a=package&feed=dotnet-eng&package=Microsoft.DotNet.Helix.Client&protocolType=NuGet&version=5.0.0-beta.20301.1

Reply:
Awesome, will take a look at the package.